src/components/multi-select/multi-select.css

```diff
@@ -0,0 +1,80 @@
+@import '../../styles/colors-v2.css';
+@import '../../styles/spacing.css';
+
+.container {
+  composes: tag-list from '../tag-list/tag-list.css';
+  composes: input-container from '../__internal__/abstract-custom-field/abstract-custom-field.css';
+  height: auto;
+  padding-top: 0;
+  padding-bottom: 0;
+}
+
+.icon-clear {
+  cursor: pointer;
+  padding: var(--spacing-small);
+}
+
+.icon-clear-readonly {
+  composes: icon-clear;
+  cursor: unset;
+  pointer-events: none;
+}
+
+.icon-container {
+  composes: icons from '../form-control-icons/form-control-icons.css';
+  top: calc(var(--spacing-large) + var(--spacing-small));
```

Comment:
Note: This is necessary to keep the icons in the top-right-hand corner of the input.